### ISSUE TITLE
'SilverStripe\TagField\TagFieldReadonly' not found

### ISSUE THREAD

**Issue by axllent** [silverstripe/silverstripe-tagfield#99]
I'm using SS4 so using the current master branch.

TagField's `performReadonlyTransformation()` refers to `TagFieldReadonly` which doesn't exist. This is problematic when trying to view an object's history tab for instance (eg: a Page/BlogPage etc that uses the TagField).

Either `SilverStripe\TagField\TagField\Readonly` needs to be renamed, or TagField's `performReadonlyTransformation()` should refer to the correct class.

**Comment by robbieaverill** [comment]
Ok, so the reference in `TagField::performReadonlyTransformation` needs to be `TagField\Readonly::class` or import `SilverStripe\TagField\TagField\Readonly` at the top and use `Readonly::class`.

Ideally we'd add a unit test covering this too, e.g.:

```php
public function testReadonlyTransformationReturnsReadonlyField()
{
    $field = new TagField('Test');
    $this->assertInstanceOf(Readonly::class, $field->performReadonlyTranformation());
}
```

If you have some time and would like to make a PR with the changes, that'd be awesome :)

You _may_ run into some difficulty with the source list, maybe. I did when I spent 30 seconds testing it earlier 😆

**Comment by axllent** [comment]
@robbieaverill Yeah, I'm also hitting the same issue .... `Fatal error: $source passed in as invalid type in ...../vendor/silverstripe/framework/src/Forms/SelectField.php on line 176` but I'm not sure where to start looking here unfortunately.

**Comment by Friksel** [comment]
same error message as @axllent here after changing  performReadonlyTransformation() to use the right Readonly class: `Fatal error: $source passed in as invalid type in ...../vendor/silverstripe/framework/src/Forms/SelectField.php on line 176`

**Comment by dhensby** [comment]
fixed with #106
